src/Events/StatusChange.js

```diff
@@ -13,6 +13,8 @@ class StatusChange {
         this.extraData = statusChange.extraData ? statusChange.extraData : null
         this.holdToken = statusChange.holdToken ? statusChange.holdToken : null
         this.origin = statusChange.origin
+        this.isPresentOnChart = statusChange.isPresentOnChart ? statusChange.isPresentOnChart : true
```

Comment:
Don't think this check is needed: the server always sends a boolean value

Reply:
correct